gen/dcos-config.yaml

```diff
@@ -2686,3 +2686,52 @@ package:
       DCOS_UI_UPDATE_DIST_PATH=/var/lib/dcos/dcos-ui-update-service/dist
       DCOS_UI_UPDATE_DIST_LINK=/var/lib/dcos/dcos-ui-update-service/dist/ui
       DCOS_UI_UPDATE_STAGE_LINK=/var/lib/dcos/dcos-ui-update-service/dist/new-ui
+  - path: /etc/fluent-bit.env
+    content: |
+      FLUENT_BIT_CONFIG_FILE=/opt/mesosphere/etc/fluent-bit/fluent-bit.conf
+      FLUENT_BIT_PARSER_FILE=/opt/mesosphere/etc/fluent-bit/parsers.conf
+  - path: /etc/fluent-bit/fluent-bit.conf
+    content: |
+      [SERVICE]
+          # Flush
+          # =====
+          # Set an interval of seconds before to flush records to a destination
+          Flush        5
+          # Daemon
+          # ======
+          # Instruct Fluent Bit to run in foreground or background mode.
+          Daemon       Off
+          # Log_Level
+          # =========
+          # Set the verbosity level of the service, values can be:
+          #
+          # - error
+          # - warning
+          # - info
+          # - debug
+          # - trace
+          #
+          # By default 'info' is set, that means it includes 'error' and 'warning'.
+          Log_Level    info
+          # Parsers_File
+          # ============
+          # Specify an optional 'Parsers' configuration file
+          Parsers_File parsers.conf
+          Plugins_File plugins.conf
+          # HTTP Server
+          # ===========
+          # Enable/Disable the built-in HTTP Server for metrics
+          HTTP_Server  Off
+          HTTP_Listen  0.0.0.0
```

Comment:
Since `HTTP_Server Off`, I wonder if it would be cleaner to comment the two remaining HTTP_* parameters: `HTTP_Listen` and `HTTP_Port`